frontend/integration-tests/tests/performance.scenario.ts

```diff
@@ -32,7 +32,7 @@ const chunkedRoutes = OrderedMap<string, {section: string, name: string}>()
   .set('limit-range', {section: 'Administration', name: 'Limit Ranges'})
   .set('custom-resource-definition', {section: 'Administration', name: 'CRDs'})
   .set('catalog', {section: 'Catalog', name: 'Developer Catalog'})
-  .set('operator-hub', {section: 'Catalog', name: 'OperatorHub'});
+  .set('operator-hub', {section: 'Catalog', name: 'Operator Hub'});
```

Comment:
This was recently renamed. I think what's in master is correct

Reply:
so, my cluster is old?  so I should revert this change?

Reply:
Yeah, you should take out this change